Story/AriapetQuests.cs

```diff
@@ -0,0 +1,43 @@
+//cs_include Scripts/CoreBots.cs
+//cs_include Scripts/CoreStory.cs
+using Skua.Core.Interfaces;
+
+public class AriapetQuests {
```

Comment:
if the quests are all in the same map name the file  and main class according to the map name "AriaPet.cs" / class AriaPet
and so on